tests/live.rs

```diff
@@ -106,10 +106,19 @@ fn get_blocker_engine_deserialized() -> Engine {
     use futures::FutureExt;
     let async_runtime = Runtime::new().expect("Could not start Tokio runtime");
 
+    let brave_service_key = std::env::var("brave_service_key")
```

Comment:
can we make this uppercase? GitHub only sets them so.

Reply:
Yeah, that's fine too. Was just copying the format from brave-browser `.npmrc`

Reply:
still doesn't find it